### DISCUSSION TITLE
Feature Request: wish regexp_substr

### DISCUSSION THREAD

**Discussion by wonb168** [duckdb/duckdb#6261] [Ideas]
I want get columns part from create table sql such as
```
create table test ( a int, b text) distribute by (a)
```
I want get `a int, b text`, In postgres it can:
```
select substring('create table test (a int, b text) distributed by (a)' from '\((.*)\) distributed')
```
but how to do this in duckdb?

**Comment by Mause** [comment]
You can probably do this using string_split_regex(string, regex)?
